readthedocs/storage/s3_storage.py

```diff
@@ -12,7 +12,7 @@
 from django.conf import settings
 from django.contrib.staticfiles.storage import ManifestFilesMixin
```

Comment:
I think this is not used anymore, is that correct? In that case, we can delete this import from here.

Reply:
Yeah, I missed that. I deleted it.